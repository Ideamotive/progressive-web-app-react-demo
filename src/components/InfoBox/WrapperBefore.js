import React from 'react';
import styled from 'styled-components';

export default styled.div`
    &:before {
        display: inline-block;
        width: 0;
        height: 0;
        border-style: solid;
        border-width: 10px 10px 0 10px;
        border-color: #6980fe transparent transparent transparent;
        line-height: 0;
        _border-color: #6980fe #000000 #000000 #000000;
        _filter: progid:DXImageTransform.Microsoft.Chroma(color='#000000');
    }
`;