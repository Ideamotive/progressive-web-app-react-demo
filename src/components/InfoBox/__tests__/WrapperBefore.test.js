import React from 'react';
import * as renderer from 'react-test-renderer';

import WrapperBefore from '../WrapperBefore';

describe('<WrapperBefore/> component', () => {
  test('should match to snapshot', () => {
    const tree = renderer
      .create(
        <WrapperBefore/>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});

