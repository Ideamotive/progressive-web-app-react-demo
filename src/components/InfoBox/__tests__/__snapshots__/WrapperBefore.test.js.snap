// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WrapperBefore/> component should match to snapshot 1`] = `
<div
  className="sc-bdVaJa fABuTX"
/>
`;
